dap_aria_mapping/getters/taxonomies.py

```diff
@@ -139,22 +129,28 @@ def get_semantic_taxonomy(
 
 
 def get_topic_names(
-    taxonomy_class: str, name_type: str, level: int, large: bool = False
+    taxonomy_class: str,
+    name_type: str,
+    level: int,
+    long: bool = False,
+    n_top: int = None,
 ) -> Dict[str, str]:
     """Downloads topic names from S3 and returns them as a dictionary.
 
     Args:
         taxonomy_class (str): The type of taxonomy to download.
         name_type (str): The type of name to download.
         level (int): The level of the taxonomy to download.
+        n_top (int, optional): The number of top entities used to label. Defaults to None (10).
+            chatgpt name_type uses 35 entities per topic to hit the API endpoint.
 
     Returns:
         pd.DataFrame: A dictionary containing the topic names.
     """
-    if large:
+    if n_top is not None:
         return download_obj(
             BUCKET_NAME,
-            f"outputs/topic_names/large_topic_names/class_{taxonomy_class}_nametype_{name_type}_level_{str(level)}.json",
```

Comment:
Similarly to my comment in the make_topic_names -  would store these within subfolders rather than having a really complex file name